packages/bot-skeleton/src/scratch/blocks/Binary/After Purchase/trade_again.js

```diff
@@ -9,7 +9,49 @@ Blockly.Blocks.trade_again = {
     },
     definition() {
         return {
-            message0: localize('Trade again'),
+            message0: localize('Trade Again {{ checked }} {{ stop_loss_label }} {{ stop_loss }}', {
+                checked: '%1',
+                stop_loss_label: '%2',
+                stop_loss: '%3',
+            }),
+            message1: localize('{{ checked }} {{ take_profit_label }} {{ take_profit }}', {
+                checked: '%1',
+                take_profit_label: '%2',
+                take_profit: '%3',
+            }),
+            args0: [
+                {
+                    type: 'field_image_checkbox',
+                    name: 'SL_ENABALED',
+                    checked: true,
+                },
+                {
+                    type: 'field_label',
+                    name: 'SL_LABEL',
+                    text: 'Until stop loss is:',
+                },
+                {
+                    type: 'input_value',
+                    name: 'STOP_LOSS',
+                },
+            ],
+            args1: [
+                {
+                    type: 'field_image_checkbox',
+                    name: 'TP_ENABALED',
+                    checked: true,
+                },
+                {
+                    type: 'field_label',
+                    name: 'TP_LABEL',
+                    text: 'Until take profit is:',
+                },
+                {
+                    type: 'input_value',
+                    name: 'TAKE_PROFIT',
+                },
+            ],
+
```

Comment:
is it possible to enable and disable the input fields by the value of checkboxes?

Reply:
yes I am working on it